Timer/js/index.js

```diff
@@ -0,0 +1,187 @@
+/**
+ * clear object value
+ * @param {object} element
+ */
+function clear(element) {
+  element.value = '';
+}
+
+/**
+ * disable object
+ * @param {object} element
+ */
+function disable(element) {
+  element.setAttribute('disabled', 'disabled');
+}
+
+/**
+ * enable object
+ * @param {object} element
+ */
+function enabled(element) {
+  element.removeAttribute('disabled');
+}
+
+/**
+ * loop function from parameter
+ * @param {string} func
+ */
+function loop(func) {
+  for (let key in window.get) {
+
+    if (window.get.hasOwnProperty(key)) {
+      window[func](window.get[key]);
+    }
+
+  }
+}
+
+/**
+ * get seconds count
+ * @returns {number}
+ */
+function getTime() {
+  let time = 0;
+  let toSeconds = 3600;
+
+  for (let key in window.get) {
+
+    if (window.get.hasOwnProperty(key)) {
+      time += (parseInt(window.get[key].value) * toSeconds);
+      toSeconds /= 60;
+    }
+
+  }
+
+  return time;
+}
+
+/**
+ * add zeros to result
+ */
+function addZero() {
```

Comment:
You can replace your method by:
```('0' + value).slice(-2)```
and you'll have always 0 before number if your number less that 9